config/locales/en.yml

```diff
@@ -572,18 +572,18 @@ en:
       more: More on GOV.UK
       popular_links_heading: Popular on GOV.UK
       popular_links:
-        - text: 'HMRC account: sign in or set up'
-          href: /log-in-register-hmrc-online-services
-        - text: 'Universal Credit account: sign in'
-          href: /sign-in-universal-credit
-        - text: 'Personal tax account: sign in or set up'
-          href: /personal-tax-account
-        - text: 'Self Assessment tax return: sign in'
-          href: /log-in-file-self-assessment-tax-return
-        - text: 'Childcare account: sign in'
-          href: /sign-in-childcare-account
-        - text: 'Check your State Pension forecast'
-          href: /check-state-pension
+        - title: 'HMRC account: sign in or set up'
+          url: /log-in-register-hmrc-online-services
+        - title: 'Universal Credit account: sign in'
+          url: /sign-in-universal-credit
+        - title: 'Personal tax account: sign in or set up'
+          url: /personal-tax-account
+        - title: 'Self Assessment tax return: sign in'
+          url: /log-in-file-self-assessment-tax-return
+        - title: 'Childcare account: sign in'
+          url: /sign-in-childcare-account
+        - title: 'Visit the UK on a Standard Visitor visa'
```

Comment:
is this for an update to existing live popular links?

Reply:
Let me check main and rebase if necessary. We don't want to update any links in this PR.

Reply:
This should now be fixed.

Reply:
Yikes good spot Syed! 😅